app/forms/mappings/Mappings.scala

```diff
@@ -219,7 +219,7 @@ trait Mappings extends Formatters with Constraints {
         case (mapping, (regex, key)) =>
           mapping.verifying(verify[String](key, _.filterNot(_.isWhitespace).matches(regex), args: _*))
       }
-      .verifying(verify[String](maxLengthErrorKey, _.filterNot(_.isWhitespace).length <= maxLength, args: _*))
+      .verifying(verify[String](maxLengthErrorKey, _.length <= maxLength, args: _*))
```

Comment:
The same logic is present below in `nonRequiredPostCode` method (used in land or property uk manual entry address page). It might not be a problem like this due to the additional transformation happening on the postcoode, but it might be worthwhile to double check.

Reply:
I've just double checked `nonRequiredPostCode` and it's working as it should